custom_components/stellantis_vehicles/stellantis.py

```diff
@@ -487,7 +486,8 @@ def _on_mqtt_message(self, client, userdata, msg):
 
     async def send_mqtt_message(self, service, message, vehicle, store=True):
         _LOGGER.debug("---------- START send_mqtt_message")
-        await self.refresh_mqtt_token(force=(store == False))
+        #TODO: check/confirm if a mqtt refresh_token is really needed here
+        await self.refresh_tokens(force=(store == False))
```

Comment:
@andreadegiovine your commit replaced refresh_mqtt_token() with refresh_tokens() .....
That's why commands are not working in 0.4.0 :-(